cli/src/commands/validate/validate.ts

```diff
@@ -3,39 +3,56 @@ import { existsSync, promises as fs, readFileSync, readdirSync, statSync } from
 import pkg from '@stoplight/spectral-core';
 const { Spectral } = pkg;
 import { getRuleset } from '@stoplight/spectral-cli/dist/services/linter/utils/getRuleset.js';
+import * as winston from 'winston';
 
-export default async function validate(jsonSchemaInstantiationLocation: string, jsonSchemaLocation: string, metaSchemaPath: string) {
+let logger: winston.Logger; // defined later at startup
+
+export default async function validate(jsonSchemaInstantiationLocation: string, jsonSchemaLocation: string, metaSchemaPath: string, debug: boolean) {
+    initLogger(debug);
```

Comment:
should we be passing around wether we are in debug mode or not? seems like this should be some sort of global variable, or dependency injected? This could be different because it's node or CLI tool